packages/libs/src/api/entityManager.ts

```diff
@@ -143,7 +144,8 @@ export class EntityManager extends Base {
         is_private: playlist.is_private,
         is_image_autogenerated: false,
         is_stream_gated: playlist.is_stream_gated,
-        stream_conditions: playlist.stream_conditions
+        stream_conditions: playlist.stream_conditions,
+        release_date: playlist.release_date
```

Comment:
should we be setting is_scheduled_release here too?

Reply:
yeah I don't have enough context here

Reply:
is_scheduled_release is also set through metadata here:
https://github.com/AudiusProject/audius-protocol/blob/main/packages/discovery-provider/src/tasks/entity_manager/entities/track.py#L406-L412

> afaict this is for interfacing with libs, and is_scheduled_release is set in indexing based on whether the release date is in the future or not.

where does this happen? it's tricky to rely on indexing to determine if something is a scheduled release because a future time for the client may not be a future time at indexing. the user could hang on the page for a while or a node could be down. there were some issues w this for track scheduled releases so that's why i added the is_scheduled_release to be set by client and let indexing know for sure.

Reply:
ah good point ok, i will add it.

Reply:
and you're right i actually did not set it.. my b